addons/binding/org.openhab.binding.irtrans/src/main/java/org/openhab/binding/irtrans/handler/TransceiverStatusListener.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.irtrans.handler;
+
+import org.openhab.binding.irtrans.IrCommand;
+
+/**
+ * The {@link TransceiverStatusListener} is interface that is to be implemented
+ * by all classes that wish to be informed of events happening to a infrared
+ * transceiver
+ *
+ * @author Karel Goderis - Initial contribution
+ * @since 2.1.0
+ *
+ */
+public interface TransceiverStatusListener {
+
+    /**
+     *
+     * Called when the ethernet transceiver/bridge receives an infrared command
+     *
+     * @param bridge
+     * @param command - the infrared command
+     */
+    public void onCommandReceived(EthernetBridgeHandler bridge, IrCommand command);
+
+    /**
+     *
+     * Called when the connection with the remote transceiver is lost
+     *
+     * @param bridge
+     */
+    public void onBridgeDisconnected(EthernetBridgeHandler bridge);
```

Comment:
You are aware that ThingHandlers meanwhile are notified about status changes of their bridge? You might hence not need these two methods anymore or are they doing something else?